ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -244,6 +244,7 @@ class Metrics(val registry: MetricRegistry) {
       val uploadPackages: Timer = registry.timer(prefix :+ "upload_packages")
       val publishConfiguration: Timer = registry.timer(prefix :+ "publish_configuration")
 
+      // FIXME Name mushing and inconsistencies here, tracked by https://github.com/digital-asset/daml/issues/5926
```

Comment:
The issue is great; I see no need for the comment.

Reply:
To make sure that people addressing the `FIXME` in this file are aware that there already is a ticket.